textdb/textdb-dataflow/src/test/java/edu/uci/ics/textdb/dataflow/regexmatch/RegexMatcherTest.java

```diff
@@ -20,64 +26,140 @@
 	@Test
 	public void testGetNextTuplePeopleFirstName() throws Exception {
 		List<ITuple> data = TestConstants.getSamplePeopleTuples();
-		RegexMatcherTestHelper test = new RegexMatcherTestHelper(TestConstants.SCHEMA_PEOPLE, data);
+		RegexMatcherTestHelper testHelper = new RegexMatcherTestHelper(TestConstants.SCHEMA_PEOPLE, data);
 		
-		List<ITuple> expected = new ArrayList<ITuple>();
-		expected.add(data.get(0));
-		expected.add(data.get(2));
-		test.runTest("b.*", TestConstants.FIRST_NAME);
-		Assert.assertTrue(test.matchExpectedResults(expected));
+		testHelper.runTest("g[^\\s]*", TestConstants.FIRST_NAME);
+		List<ITuple> exactResults = testHelper.getResults();
 		
-		test.cleanUp();
+		List<ITuple> expectedResults = new ArrayList<ITuple>();
+		
+		//expected to match "brad lie angelina"
+		Schema spanSchema = testHelper.getSpanSchema();
+		List<Span> spans = new ArrayList<Span>();
+		spans.add(new Span(TestConstants.FIRST_NAME, 11, 17, "g[^\\s]*", "brad lie angelina"));
```

Comment:
This offsets are supposed to be equal to offsets returned by Java regex matcher. It uses "gap offsets".